muikku-core-plugins/src/main/java/fi/otavanopisto/muikku/plugins/me/GuidanceCounselorRestModel.java

```diff
@@ -11,8 +12,8 @@ public GuidanceCounselorRestModel() {
   }
   
   public GuidanceCounselorRestModel(String id, Long userEntityId, String firstName, String lastName, String email,
-      Map<String, String> properties, OrganizationRESTModel organization, String role, boolean hasImage, boolean chatAvailable) {
-    super(id, userEntityId, firstName, lastName, email, properties, organization, role, hasImage);
+      Map<String, String> properties, OrganizationRESTModel organization, List<String> roles, boolean hasImage, boolean chatAvailable) {
```

Comment:
No biggie but this probably would be more accurate as Set rather than List? Most likely affects several other files not mentioned separately.

Reply:
Changed to Set